operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -274,6 +275,12 @@ public NextAction apply(Packet packet) {
       if (topologyYaml != null) {
         LOGGER.fine("topology.yaml: " + topologyYaml);
         DomainTopology domainTopology = parseDomainTopologyYaml(topologyYaml);
+        if (!domainTopology.getValidationErrors().isEmpty()) {
+          for (String err : domainTopology.getValidationErrors()) {
+            LOGGER.severe(err);
```

Comment:
Throw an exception with the errors.  This will log as severe and trigger the domain processing retry logic.

Reply:
We don't want to trigger any retries when there's validation errors during introspection. validation errors are due to the domain being invalid, we need to stop processing and log this as severe, so that customer can fix the issue with the domain.